R/init.R

```diff
@@ -36,11 +36,8 @@
 #'   The header of the app.
 #' @param footer (`shiny.tag` or `character(1)`)
 #'   The footer of the app.
-#' @param id (`character`)
-#'   module id to embed it, if provided,
-#'   the server function must be called with [shiny::moduleServer()];
-#'   See the vignette for an example. However, [ui_teal_with_splash()]
-#'   is then preferred to this function.
+#' @param id optional (`character`)
+#'   specifying the module id to be used when embedding the teal app within another `shiny` app as a module.
```

Comment:
```suggestion
#' @param id (`character`) 
#'   Optional string specifying the `shiny` for `teal` in cases it is used as a module 
#'   rather than a standalone `shiny` app. This is a legacy feature.
```